manifests/params.pp

```diff
@@ -4,8 +4,17 @@
 # Set variables for names and paths
 #
 class dhcp::params {
+  case $::osfamily {
+    /RedHat/: {
```

Comment:
Changed to quoted string. I tend to use regex in case because I'm always nervous about case issues :)